sentry-sample/src/main/java/io/sentry/sample/MainActivity.java

```diff
@@ -25,35 +24,26 @@ protected void onCreate(Bundle savedInstanceState) {
             });
 
     findViewById(R.id.send_message)
-        .setOnClickListener(
-            view -> {
-              Sentry.captureMessage("Some message.");
-            });
+        .setOnClickListener(view -> Sentry.captureMessage("Some message."));
 
     findViewById(R.id.capture_exception)
         .setOnClickListener(
-            view -> {
-              Sentry.captureException(
-                  new Exception(new Exception(new Exception("Some exception."))));
-            });
+            view ->
+                Sentry.captureException(
+                    new Exception(new Exception(new Exception("Some exception.")))));
 
     findViewById(R.id.breadcrumb)
         .setOnClickListener(
             view -> {
-              Sentry.configureScope(
-                  scope -> {
-                    Breadcrumb breadcrumb = new Breadcrumb();
-                    breadcrumb.setMessage("Breadcrumb");
-                    scope.addBreadcrumb(breadcrumb);
-                    scope.setExtra("extra", "extra");
-                    scope.setFingerprint(Collections.singletonList("fingerprint"));
-                    scope.setLevel(SentryLevel.INFO);
-                    scope.setTransaction("transaction");
-                    User user = new User();
-                    user.setUsername("username");
-                    scope.setUser(user);
-                    scope.setTag("tag", "tag");
-                  });
+              Sentry.addBreadcrumb("Breadcrumb");
```

Comment:
We could demo here the `beforeBreadcrumb` dropping a crumb too. Would be nice.

Reply:
it should be done on `MyApplication` though.